oe_theme.theme

```diff
@@ -1522,3 +1523,84 @@ function oe_theme_preprocess_page(array &$variables): void {
     'id' => ['main-content'],
   ]);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_oe_organisation(&$variables): void {
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+
+  $entity = &$variables['entity'];
+  // Prepare fields.
+  $fields_values = ['name' => 'name', 'oe_acronym' => 'acronym'];
+  foreach ($fields_values as $field => $pattern_key) {
+    $variables[$pattern_key] = '';
+    if (!$entity->get($field)->isEmpty()) {
+      $variables[$pattern_key] = $entity->get($field)->value;
+    }
```

Comment:
The first line of the for loop was executed for nothing of the if condition has met. I like to do this in for loops, instead of negating the isEmpty() to make it more readable:
```suggestion
    if ($entity->get($field)->isEmpty()) {
      $variables[$pattern_key] = '';
      continue;
    }
    $variables[$pattern_key] = $entity->get($field)->value;
```

Reply:
done